run-precommit-checks.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Should we have this script portable instead?

Reply:
OS-wise

Reply:
We are already using bash in setup.sh, so I guess it should be OK here as well.